metrics.py

```diff
@@ -331,7 +331,7 @@ def ingest_release_schedule(project):
 
         # Extract Factory "release schedule" from Tumbleweed snapshot list.
         command = 'rsync rsync.opensuse.org::opensuse-full/opensuse/tumbleweed/iso/Changes.* | ' \
-            'grep -oP "Changes\.\K\d{5,}"'
+            'grep -oP "' + r'Changes\.\K\d{5,}' + '"'
```

Comment:
this isn't really a python regexp, so why mark it?

Reply:
https://docs.python.org/2.0/ref/strings.html

```
Unlike Standard C, all unrecognized escape sequences are left in the string unchanged, i.e., the backslash is left in the string.
```

Which is why it works, but flake just warns about it. We can choose to ignore those as well, but seems like a decent best practice.

Reply:
so r is for raw not for regexp? Wow :)

Reply:
Yeah, the way it is used is basically for regex. hehe :)